docs/contributor/04-10-ci-cd.md

```diff
@@ -6,19 +6,21 @@ The following CI jobs are part of the development cycle. They verify the functio
 
 | Name | Required | Description |
 |------|----------|-------------|
-|[`pre-keda-manager-operator-build`](https://github.com/kyma-project/test-infra/blob/main/templates/data/generic_module_data.yaml#L144)|true|Builds Keda operator's image and pushes it to the `dev` registry.|
-|[`pre-keda-module-build`](https://github.com/kyma-project/test-infra/blob/main/templates/data/generic_module_data.yaml#L102)|true|Builds module's OCI image and pushes it to the `dev` artifact registry. Renders ModuleTemplate for the Keda module that allows for manual integration tests against Lifecycle Manager.|
-|[`pre-keda-manager-operator-tests`](https://github.com/kyma-project/test-infra/blob/main/templates/data/generic_module_data.yaml#L127)|true|Executes basic create/update/delete functional tests of Keda Manager's reconciliation logic.|
-|[`pre-main-keda-manager-verify`](https://github.com/kyma-project/test-infra/blob/main/templates/data/generic_module_data.yaml#L175)|true|Installs Keda Manager, not using Lifecycle Manager, and applies the sample Keda CR in a k3d cluster. Executes smoke integration test of KEDA.  |
-|[`pre-keda-manager-operator-lint`](https://github.com/kyma-project/test-infra/blob/main/templates/data/generic_module_data.yaml#L61)|false|Is responsible for linting, static code analysis.|
+|[`Keda verify / lint`](https://github.com/kyma-project/keda-manager/blob/main/.github/workflows/keda-verify.yml#L19)|true|Is responsible for the Operator linting and static code analysis.|
+|[`PR Markdown Link Check / markdown-link-check (pull_request)`](https://github.com/kyma-project/keda-manager/blob/main/.github/workflows/pr-markdown-link-check.yaml)|true|Checks if there are no broken links in the pull request `.md` files.|
+|[`Keda verify / unit-test`](https://github.com/kyma-project/keda-manager/blob/main/.github/workflows/keda-verify.yml#L29)|true|Runs basic unit tests of Operator's logic.|
+|[`Keda verify / integration-test`](https://github.com/kyma-project/keda-manager/blob/main/.github/workflows/keda-verify.yml#L38)|true|Runs the basic functionality integration test suite for the Operator in a k3d cluster.|
```

Comment:
What `Operator` do you mean? Is it `Operator` or `the Operator`?

Reply:
It's Keda Operator, so I think it's `the`

Reply:
Actually, should be without `the` :D I've fixed that :)